src/uct/ib/base/ib_device.h

```diff
@@ -56,6 +56,9 @@
 #define UCT_IB_DEVICE_SYSFS_GID_TYPE_FMT  UCT_IB_DEVICE_SYSFS_GID_ATTR_PFX "/types/%d"
 #define UCT_IB_DEVICE_SYSFS_GID_NDEV_FMT  UCT_IB_DEVICE_SYSFS_GID_ATTR_PFX "/ndevs/%d"
 
+#define UCT_IB_PCI_BUS_ID_LEN             7   /* PCI BUS ID */
```

Comment:
maybe use NAME_MAX constant?

Reply:
@yosefe We need the real length of 0000:03 which is 7 to memcpy 0000:03 from 0000:03:00.0.

Reply:
@yosefe Or is it better to use something like strstr(":") to get the length of  0000:03 then we don't need to define monstant here?

Reply:
better parse it to components

Reply:
@yosefe Maybe no need to do this because the pci bus id and format is pretty fixed. No need to add more code and logic to parse it.

Reply:
@yosefe Fixed.